### ISSUE TITLE
Add new "wound" term?

### ISSUE THREAD

**Issue by ddooley** [obophenotype/human-phenotype-ontology#4912]
I'm not sure if that is too colloquial.

**Preferred term label:**
wound

**Synonyms**

**Definition (free text, please give PubMed ID)**
A wound is a type of injury which happens relatively quickly in which skin is torn, cut, or punctured (an open wound), or where blunt force trauma causes a contusion (a closed wound). In pathology, it specifically refers to a sharp injury which damages the Epidermis of the skin.
https://en.wikipedia.org/wiki/Wound

**Parent term (use hpo.jax.org/app)**
Possibly "Abnormality of the skin" http://purl.obolibrary.org/obo/HP_0000951 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
'skin injury' [http://purl.obolibrary.org/obo/NCIT_C99062] 
or Bruise [http://purl.obolibrary.org/obo/NCIT_C37921]
or 'Puncture Wound' [IRI: http://purl.obolibrary.org/obo/NCIT_C34948]

**Your nano-attribution (ORCID)**
Not really my defn, but: 
https://orcid.org/0000-0002-8844-9165

**Comment by pnrobinson** [comment]
Thanks for the suggestion, I agree that it would be good to add a wound term to HPO. The primary issue is that there are acute wounds produced by injuries and chronic wounds such as decubitus ulcers. Therefore, this definition is not 100% correct. I wound suggest we have
wound
--acute wound
--chronic wound
See https://escholarship.org/content/qt2ng2j1m0/qt2ng2j1m0.pdf

**Comment by pnrobinson** [comment]
adding these terms.
